src/ui/components/ProposalCall.tsx

```diff
@@ -55,12 +55,14 @@ function SubCall({subCall}: SubCallProps) {
   return (
     <Card mode="outlined" style={styles.subCallContainer}>
       <Card.Content>
-        {subCall.meta && (
-          <>
-            <Paragraph>{subCall.meta}</Paragraph>
-            <Padder scale={0.5} />
-          </>
-        )}
+        <Text>
+          {subCall.meta && (
```

Comment:
This is not safe in react native, it is better to use `{subCall.meta ? <>...</> : null}`. I will add an eslint plugin to avoid this in a separate PR.

Reply:
Yo can remove the `Text` wrapper